metadata/RecommendedReviewers

```diff
@@ -0,0 +1 @@
+../.github/CODEOWNERS
```

Comment:
IIRC,symlink in Windows + subversion do not work unfortunately.

https://stackoverflow.com/questions/4056092/what-happens-if-i-add-a-symbolic-link-to-subversion

> Although Windows now supports symbolic links, and those symbolic links are compatible with Unix/Linux/Mac symbolic links, Subversion itself does not support symbolic links on Windows. I have no idea why this is the case.

Should we add this symbolic link after completely retiring subversion?

Reply:
Oh, yeah, probably.